manifests/env.pp

```diff
@@ -6,21 +6,21 @@
 # parameter
 
 define poudriere::env (
-  $ensure           = 'present',
-  $makeopts         = [],
-  $makefile         = nil,
-  $version          = '10.0-RELEASE',
-  $arch             = 'amd64',
-  $jail             = $name,
-  $paralleljobs     = $facts['processors']['count'],
-  $pkgs             = [],
-  $pkg_file         = nil,
-  $pkg_makeopts     = {},
-  $pkg_optsdir      = nil,
-  $portstree        = 'default',
-  $cron_enable      = false,
-  $cron_always_mail = false,
-  $cron_interval    = { minute => 0, hour => 0, monthday => '*', month => '*', weekday => '*' },
+  Enum['present', 'absent'] $ensure           = 'present',
+  Array[String]             $makeopts         = [],
+  Optional[String::Abspath] $makefile         = undef,
+  String                    $version          = '10.0-RELEASE',
```

Comment:
nope… this should default to the Host OS version
or is that also something you'll fix in a new round?

Reply:
if it should be changed, it should happen in another  PR, as this one only focuses on data types.

Reply:
I opened a PR with a list of the weird things I stumbled on and can help fixing.  This item is in the list!